lint CI error

Current lint target in CI fail: 
https://travis-ci.org/benoitc/gunicorn/jobs/590535074

```
$ python --version
188Python 3.7.1
189$ pip --version
190pip 19.0.3 from /home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/pip (python 3.7)
install
1911.38s$ pip install tox
21723.06s$ tox --recreate
218lint recreate: /home/travis/build/benoitc/gunicorn/.tox/lint
219lint installdeps: pylint
220lint develop-inst: /home/travis/build/benoitc/gunicorn
221lint installed: astroid==2.3.0,-e git+https://github.com/benoitc/gunicorn.git@bc6b2fd320bc70335a0f5597e86ec65429d015a0#egg=gunicorn,isort==4.3.21,lazy-object-proxy==1.4.2,mccabe==0.6.1,pylint==2.4.1,six==1.12.0,typed-ast==1.4.0,wrapt==1.11.2
222lint run-test-pre: PYTHONHASHSEED='284302499'
223lint run-test: commands[0] | pylint -j0 gunicorn tests/test_arbiter.py tests/test_config.py tests/test_http.py tests/test_invalid_requests.py tests/test_logger.py tests/test_pidfile.py tests/test_sock.py tests/test_ssl.py tests/test_statsd.py tests/test_systemd.py tests/test_util.py tests/test_valid_requests.py
224************* Module gunicorn.util
225gunicorn/util.py:326:8: W0127: Assigning the same variable 'module' to itself (self-assigning-variable)
226************* Module gunicorn.reloader
227gunicorn/reloader.py:50:16: R1724: Unnecessary "elif" after "continue" (no-else-continue)
228************* Module gunicorn.http.message
229gunicorn/http/message.py:240:12: R1723: Unnecessary "elif" after "break" (no-else-break)
230************* Module gunicorn.workers.gthread
231gunicorn/workers/gthread.py:154:12: R1723: Unnecessary "else" after "break" (no-else-break)
232
233-----------------------------------
234Your code has been rated at 9.99/10
235
236ERROR: InvocationError for command /home/travis/build/benoitc/gunicorn/.tox/lint/bin/pylint -j0 gunicorn tests/test_arbiter.py tests/test_config.py tests/test_http.py tests/test_invalid_requests.py tests/test_logger.py tests/test_pidfile.py tests/test_sock.py tests/test_ssl.py tests/test_statsd.py tests/test_systemd.py tests/test_util.py tests/test_valid_requests.py (exited with code 12)
237___________________________________ summary ____________________________________
238ERROR:   lint: commands failed
239The command "tox --recreate" exited with 1.
cache.2
240store build cache
252
253
254```

--- a/.pylintrc
+++ b/.pylintrc
@@ -21,6 +21,7 @@ disable=
     eval-used,
     fixme,
     import-error,
+    import-outside-toplevel,
     import-self,
     inconsistent-return-statements,
     invalid-name,

--- a/gunicorn/http/message.py
+++ b/gunicorn/http/message.py
@@ -242,7 +242,7 @@ class Request(Message):
                 if idx > limit > 0:
                     raise LimitRequestLine(idx, limit)
                 break
-            elif len(data) - 2 > limit > 0:
+            if len(data) - 2 > limit > 0:
                 raise LimitRequestLine(len(data), limit)
             self.get_data(unreader, buf)
             data = buf.getvalue()

--- a/gunicorn/reloader.py
+++ b/gunicorn/reloader.py
@@ -2,6 +2,7 @@
 #
 # This file is part of gunicorn released under the MIT license.
 # See the NOTICE for more information.
+# pylint: disable=no-else-continue
 
 import os
 import os.path

--- a/gunicorn/util.py
+++ b/gunicorn/util.py
@@ -323,7 +323,7 @@ def write_error(sock, status_int, reason, mesg):
 def import_app(module):
     parts = module.split(":", 1)
     if len(parts) == 1:
-        module, obj = module, "application"
+        obj = "application"
     else:
         module, obj = parts[0], parts[1]
 

--- a/gunicorn/workers/gthread.py
+++ b/gunicorn/workers/gthread.py
@@ -9,6 +9,7 @@
 # Keepalive connections are put back in the loop waiting for an event.
 # If no event happen after the keep alive timeout, the connection is
 # closed.
+# pylint: disable=no-else-break
 
 import concurrent.futures as futures
 import errno

